components/Portfolio/index.js

```diff
@@ -99,40 +115,39 @@ class PortfolioPage extends React.Component{
             </ButtonGroup>
           </PortfolioPageNavButtons>
           <ValueDisplay
-            text="Total Portfolio Value"
-            value={formatMonetaryValue(assets.length > 0 ? assets[assets.length - 1].totalAssetValue : 0)}
-            icon={<PieChart />}
+            text={currentView === PortfolioTypes.MANAGED_ASSETS ? 'Total Portfolio Revenue' : 'Total Portfolio Value'}
+            value={formatMonetaryValue(currentView === PortfolioTypes.MANAGED_ASSETS ? stats.totalAssetRevenue : stats.totalAssetValue)}
+            icon={currentView === PortfolioTypes.MANAGED_ASSETS ? <LineChart /> : <PieChart />}
+            hasSeparator
+            hasIcon
+            hasShadow
+            isBlue={currentView === PortfolioTypes.INVESTMENTS}
+            isGreen={currentView === PortfolioTypes.MANAGED_ASSETS}
+            coloredBackground
+            customClassName="PortfolioPage__ValueDisplay--is-1"
+          />
+          <ValueDisplay
+            text="Unrealised Profit"
+            value={formatMonetaryValue(currentView === PortfolioTypes.MANAGED_ASSETS ? unrealisedProfitManager : unrealisedProfitInvestor)}
+            icon={<UnrealisedProfit />}
             hasSeparator
             hasIcon
             hasShadow
             isBlue
             coloredBackground
-            customClassName="PortfolioPage__ValueDisplay--is-portfolioRevenue"
+            customClassName="PortfolioPage__ValueDisplay--is-2"
```

Comment:
Preferred when we had names over numbers, numbers give me no context

Reply:
Same for me but because the first one can be either Revenue or Value I went with numbers. because my brain couldn't decide on a name.

Reply:
Updated!